### DISCUSSION TITLE
Hearing aids

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#602] [General]
I've been looking at hearing aids for under $1,000 and it is looking like Walmart as usual has the best buys. Anyone in here have one of these Wallyworld aids, if so what do you think of them? What's better, the in the ear or over the ear type or pros and cons of each. Or if you know of a better buy I'd appreciate the info, thanks.

**Comment by ellielo** [comment]
I know there are lots of old people in here with hearing aids damnit. Quit hiding, speak up. :lol: I have found a new site though, Costco. They have hearing testers too.

**Comment by MirkosMangle** [comment]
I can recommend you this [hearing aid supplies online](https://audiologyisland.com/online-shop/). My wife always buys hearing aids and supplies for them here. 
There you can find them high-qualitative and at an excellent price. We chose Phonak because they’re convenient and easy to use.
